app-benchmarks/cpuburn/cpuburn-1.4a-r3.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+MY_PV="${PV/./_}"
+
+DESCRIPTION="CPU testing utilities in optimized assembler for maximum loading"
+HOMEPAGE="https://downloads.bl4ckb0x.de/pages.sbcglobal.net/redelm/"
+SRC_URI="https://downloads.bl4ckb0x.de/pages.sbcglobal.net/redelm/${PN}_${MY_PV}_tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="-* ~amd64 ~arm ~x86"
+LICENSE="GPL-2"
+SLOT="0"
+
+PATCHES=( "${FILESDIR}/${P}-variables.patch" )
+
+QA_FLAGS_IGNORED="
+	usr/bin/burnBX
+	usr/bin/burnK6
+	usr/bin/burnK7
+	usr/bin/burnMMX
+	usr/bin/burnP5
+	usr/bin/burnP6
+"
+
+QA_TEXTRELS="${QA_FLAGS_IGNORED}"
+
+src_prepare() {
```

Comment:
since `src_prepare` is just doing `default`, you can remove the function entirely.

Reply:
Eh? There are two sed commands in `src_prepare`? Did I miss something completly?